README.md

```diff
@@ -1,6 +1,6 @@
 nxdk - *the new open source xdk*
 ================================
-nxdk is a software development kit for the original Xbox. nxdk is a revitalization of [OpenXDK](https://web.archive.org/web/20170624051336/http://openxdk.sourceforge.net:80/).
+nxdk is a software development kit for the original Xbox. nxdk is a revitalization of [OpenXDK](https://web.archive.org/web/20170624051336/http://openxdk.sourceforge.net:80/). It is maintained by the [XboxDev](https://github.com/XboxDev/XboxDev) community.
```

Comment:
In addition to this, we should probably add a note to the wiki, explicitly recommending the Discord channel.